test/integration/app.js

```diff
@@ -13,7 +14,7 @@ describe('app', () => {
         .get('/')
```

Comment:
What do you reckon to checking the status code of this request (can be done separately from this PR if you agree)?